docs/technical/shell-integration.md

```diff
@@ -46,22 +46,50 @@ use **Git Bash** as a reference for the rest of the process.
 ### Step 1: Find the shell executable
 
 The `getAvailableShells()` method is used to find each shell, as some may not
-be present on the user's machine. You will need to add some code in here.
+be present on the user's machine.
 
-For Git Bash we perform a couple of checks:
+This is the example for how we resolve if Git Bash is installed.
 
 ```ts
-const gitBash = await readRegistryKeySafe(
-  'HKEY_LOCAL_MACHINE\\SOFTWARE\\GitForWindows'
-)
-if (gitBash.length > 0) {
-  const installPathEntry = gitBash.find(e => e.name === 'InstallPath')
-  if (installPathEntry) {
+  const gitBashPath = await findGitBash()
```

Comment:
Just a heads up: these examples are actually based off https://github.com/desktop/desktop/pull/3975 which hasn't been merged yet. These are internals but I found it compelling enough to address now rather than later.